Doc/source/conf.py

```diff
@@ -25,40 +25,40 @@
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #
-needs_sphinx = '1.3'
+needs_sphinx = "1.3"
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.viewcode']
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.viewcode", "sphinxcontrib.napoleon"]
 
-autodoc_mock_imports = ['gtk']
+autodoc_mock_imports = ["gtk"]
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = '.rst'
+source_suffix = ".rst"
```

Comment:
Associated issue report https://github.com/fonttools/fonttools/issues/1266

I believe that we can convert this to:

```python
source_suffix = ['.rst', '.md']
```

and provide Markdown support in additon to the rST support.  This would allow those who would like to write docs in Markdown to do so.  Unclear to me whether you can include the source parsing scripts in Markdown files.  This may only apply to rST docs?  There are probably situations that warrant free text documentation that is written from scratch. For instance, for the executables that are released with the library.  Those might be good situations for Markdown files.

Reply:
I can confirm that Github flavored Markdown files build to HTML without issues in a `*.rst` + `*.md` Sphinx doc build environment.  Must add the `recommonmark` doc build dependency and the following to the `conf.py` file:

https://github.com/chrissimpkins/fontbakery/blob/11365943499f2f3aa5ee3b3bcb3c0f757095ecf9/docs/source/conf.py#L47-L51